examples/iam-policy-management.v1.test.js

```diff
@@ -292,7 +292,37 @@ describe('IamPolicyManagementV1', () => {
       });
 
     // end-delete_policy
-  })
+  });
+  test('patchPolicy request example', done => {
```

Comment:
I think you might need to move the PATCH example/test after UPDATE like you did in Go and Python case

Reply:
Good catch.